### PR TITLE
[8.0] Fix account.move.line field last_rec_date

### PR DESCRIPTION
**Pull request by ghost**
In previous implementation of the field with the api 7.0, the field was not always computed at the right moment.
This would cause last_rec_date to be null. The drawback is that the amounts on the aged partner balance may be incorrect.
For example, if you have an invoice on May and a payment on June, the amount would not appear on the partner
aged balance of May.

### CODE REVIEW

File: account_financial_report_webkit/i18n/account_financial_report_webkit.pot [hunk 1]
```diff
@@ -1507,7 +1507,7 @@ msgstr ""
```
Comment (pedrobaeza) [inline account_financial_report_webkit/i18n/account_financial_report_webkit.pot]: You can remove this file as it's no longer needed.

**Comment by pedrobaeza** [comment]
Thanks for this improvement. Minor nitpick, but :+1:

**Comment by ghost** [comment]
@pedrobaeza I improved the migration script and removed the pot file.

File: account_financial_report_webkit/__openerp__.py [hunk 2]
```diff
@@ -169,8 +169,12 @@
 the header and footer are created as text with arguments passed to
 wkhtmltopdf. The texts are defined inside the report classes.
 """,
-    'version': '8.0.1.1.0',
```
Comment (elicoidal) [inline account_financial_report_webkit/__openerp__.py:172]: Please remove the descriptioon key and input the content in README

**Comment by elicoidal** [comment]
small non blocking detail.
:+1:

**Comment by ghost** [comment]
@elicoidal Thanks for the review. I moved the description to README.

**Pull request merged**